src/Pages/Inventory/InventoryPage.js

```diff
@@ -1,22 +1,28 @@
 import { Switch, Route } from "react-router-dom";
 import InventoryItemDetails from '../../Components/InventoryItemDetails/InventoryItemDetails'
 import PageHeader from "../../Components/Header/Header";
-import './InventoryPage.scss';
+import InventoryList from "../../Components/InventoryList/InventoryList";
+import './InventoryPage.scss'
 
 function InventoryPage()  {
 
-
-    return (<>
+    return (
+    <>
         <PageHeader isActive='Inventory' />
         <div className="component-wrapper">
                 <Switch>
+                    <Route path='/inventory' exact component={InventoryList} />
+                    <Route path='/inventory/add'  component='' />
+                    <Route path='/inventory/:id'  component='' />
```

Comment:
```suggestion

```
Suggest removing this line, as it's being interpreted first instead of line 21, and this one doesn't have a component tied to it.